pom.xml

```diff
@@ -1,21 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <!--TODO: Update once the client code is cleaned UP -->
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.11</version>
+        <version>3.14</version>
+        <relativePath />
     </parent>
 
     <artifactId>swarm-plugin</artifactId>
     <packaging>pom</packaging>
     <name>Jenkins Self-Organizing Swarm Plug-in</name>
     <version>3.13-SNAPSHOT</version>
 
+    <properties>
+        <jenkins.version>2.60.1</jenkins.version>
+        <java.level>8</java.level>
```

Comment:
No particular reason to bump, because Java 8 requirement applies only to the client side.
But I am fine with that, the version is 1 year old

Reply:
Agreed. It seemed cleaner to me to have the whole project using Java 8 rather than a mix of Java 7 and Java 8.